packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -335,13 +344,16 @@ class IconButton extends StatelessWidget {
     else
       currentColor = disabledColor ?? theme.disabledColor;
 
-    final VisualDensity effectiveVisualDensity = visualDensity ?? theme.visualDensity;
+    final VisualDensity effectiveVisualDensity =
+        visualDensity ?? theme.visualDensity;
 
-    final BoxConstraints unadjustedConstraints = constraints ?? const BoxConstraints(
-      minWidth: _kMinButtonSize,
-      minHeight: _kMinButtonSize,
-    );
-    final BoxConstraints adjustedConstraints = effectiveVisualDensity.effectiveConstraints(unadjustedConstraints);
+    final BoxConstraints unadjustedConstraints = constraints ??
+        const BoxConstraints(
+          minWidth: _kMinButtonSize,
+          minHeight: _kMinButtonSize,
+        );
+    final BoxConstraints adjustedConstraints =
+        effectiveVisualDensity.effectiveConstraints(unadjustedConstraints);
```

Comment:
same here, all these changes just seem spurious? as far as i can tell there's no change to the logic here.

Reply:
As I mentioned, VSCode auto formats, and this is what I get